source/adapters/level_zero/queue.cpp

```diff
@@ -593,11 +603,17 @@ UR_APIEXPORT ur_result_t UR_APICALL urQueueRetain(
   return UR_RESULT_SUCCESS;
 }
 
-UR_APIEXPORT ur_result_t UR_APICALL urQueueRelease(
-    ur_queue_handle_t UrQueue ///< [in] handle of the queue object to release
+UR_APIEXPORT ur_result_t UR_APICALL urQueueRetain(
```

Comment:
Aliasing API function names like this is probably going to be confusing.

Reply:
Alternatives are:
- add prefix to the function, e.g. this would become urQueueRetainImpl or something like that
- make those functions member functions of the proper handle type, i.e. we would call UrQueue->Retain()